### PR TITLE
document.SetConformance is added with an example

### PR DESCRIPTION
**Pull request by zgordan-vv**
A fix for the issue https://github.com/unidoc/unioffice/issues/372

---
This change is [Reviewable](https://reviewable.io/reviews/unidoc/unioffice/399)

### CODE REVIEW

File: _examples/document/conformance_attribute/main.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+func main() {
+	doc, err := document.Open("document.docx")
+	if err != nil {
+		panic(err)
+	}
+	doc.SetConformance(st.ST_ConformanceClassTransitional)
+	doc.SaveToFile("conformance_transitional.docx")
+	doc.SetConformance(st.ST_ConformanceClassStrict)
+	doc.SaveToFile("conformance_strict.docx")
+	doc.SetConformance(st.ST_ConformanceClassUnset)
```
Comment (gunnsth) [inline _examples/document/conformance_attribute/main.go:19]: What does this one mean?  Is it ever unset?

File: _examples/document/conformance_attribute/main.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	"github.com/unidoc/unioffice/document"
+	st "github.com/unidoc/unioffice/schema/soo/ofc/sharedTypes"
+)
+
+func main() {
+	doc, err := document.Open("document.docx")
+	if err != nil {
+		panic(err)
+	}
+	doc.SetConformance(st.ST_ConformanceClassTransitional)
```
Comment (gunnsth) [inline _examples/document/conformance_attribute/main.go:15]: Would it be nicer to just set doc.SetStrict(true)  default is transitional I think always for word files.
I am not sure what the unset option means.

File: _examples/document/conformance_attribute/main.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
```
Comment (gunnsth) [inline _examples/document/conformance_attribute/main.go:1]: 2020

**Comment by gunnsth** [comment]
Great, let's see if we get feedback for #372 so we can determine if this solves the problem.

File: _examples/document/conformance_attribute/main.go (hunk 1 above)
Comment (zgordan-vv) [reply _examples/document/conformance_attribute/main.go:19]: yes, there is such an option among others in the specification

**Pull request merged**